packages/vats/src/distributeFees.js

```diff
@@ -62,5 +62,7 @@ export async function buildDistributor(
   };
 
   const epochNotifier = E(epochTimer).makeNotifier(0n, epochInterval);
-  return observeNotifier(epochNotifier, timeObserver);
+  observeNotifier(epochNotifier, timeObserver).catch(e => {
+    console.error('fee distributor failed with', e);
```

Comment:
loud enough?

Reply:
Yep!

Reply:
more to the point... @michaelfig tells me he has plans to systematically address unhandled rejections, but it depend on work in endo...